app/policies/admin_grove_playlist_policy.rb

```diff
@@ -0,0 +1,5 @@
+class AdminGrovePlaylistPolicy < ApplicationPolicy
+  def show?
+    user.present? && user.is_a?(Teacher)
```

Comment:
oh, never mind. all teachers need access to these resources. but not all teachers are admins. fair enough.